third_party/move/move-vm/test-utils/src/storage.rs

```diff
@@ -70,82 +66,11 @@ impl TableResolver for BlankStorage {
         _handle: &TableHandle,
         _key: &[u8],
         _maybe_layout: Option<&MoveTypeLayout>,
-    ) -> Result<Option<Bytes>, PartialVMError> {
+    ) -> PartialVMResult<Option<Bytes>> {
         Ok(None)
     }
 }
 
-/// A storage adapter created by stacking a change set on top of an existing storage backend.
```

Comment:
I'm fine with this being removed for now if we have nothing depending on it, but want to point out that this still represents some useful usage patterns, particularly for simulations. 

For example, right now I'm working on local simulation of governance proposals, and in order for it to work, I need to use real network states but make modify certain states to bypass the governance process.

Reply:
Agreed! But it feels like this abstraction belongs to adapter layer - we have respawned session which does this stashing, for instance. We can keep it, no preference here, but the two use cases it had were not using it in the right way.